src/WorksomeRectorConfig.php

```diff
@@ -41,8 +42,8 @@ public static function setup(RectorConfig $rectorConfig): void
         $rectorConfig->rule(RemoveUnusedVariableInCatchRector::class);
 
         // DEAD CODE set rules
-        $rectorConfig->rule(\Rector\DeadCode\Rector\FunctionLike\RemoveCodeAfterReturnRector::class);
-        $rectorConfig->rule(\Rector\DeadCode\Rector\ClassMethod\RemoveDeadConstructorRector::class);
+        $rectorConfig->rule(RemoveUnreachableStatementRector::class);
```

Comment:
The `Rector\DeadCode\Rector\FunctionLike\RemoveCodeAfterReturnRector` rule was removed in https://github.com/rectorphp/rector-src/pull/2199, so I've replaced it with `RemoveUnreachableStatementRector` which does the same and more. 👍🏻